operations/mimir-mixin/dashboards/tenants.libsonnet

```
@@ -530,5 +530,106 @@ local filename = 'mimir-tenants.json';
         ) +
         { legend: { show: false } },
       )
-    ),
+    )
+
+    .addRow(
+      $.row('Read Path - Query Frontend Insights')
+      .addPanel(
+        local title = 'Rate of Read Requests - Query Frontend';
+        $.panel(title) +
+        $.queryPanel(
+          'sum(rate(cortex_query_frontend_queries_total{%s, container="query-frontend", user="$user"}[$__rate_interval]))' % $.namespaceMatcher(),
+          'rate'
+        )
+      )
+      .addPanel(
+        local title = 'Rate of Read Requests - Ruler Query Frontend';
+        $.panel(title) +
+        $.queryPanel(
+          'sum(rate(cortex_query_frontend_queries_total{%s, container="ruler-query-frontend", user="$user"}[$__rate_interval]))' % $.namespaceMatcher(),
+          'rate'
+        )
+      )
+    )
+    .addRow(
+      $.row('Read Path - Query Scheduler Insights')
+      .addPanel(
+        local title = 'Number of Queries Queued - Query Scheduler';
+        $.panel(title) +
+        $.queryPanel(
+          [
+            'sum(cortex_query_scheduler_queue_length{%s, container="query-scheduler", user="$user"})' % $.namespaceMatcher(),
```

```suggestion
            'sum(max_over_time(cortex_query_scheduler_queue_length{%s, container="query-scheduler", user="$user"}[$__interval])' % $.namespaceMatcher(),
```

If we use `max_over_time` then we will capture more peaks, but I am wondering what makes more sense, if we should `sum()` or `max()` over the pods.... Perhaps `sum()` is fine.

---

Lets leave it.